scarb/tests/package.rs

```diff
@@ -1525,3 +1526,105 @@ fn package_with_publish_disabled() {
             [..]Packaged [..] files, [..] ([..] compressed)
         "#});
 }
+
+#[test]
+fn package_with_package_script() {
+    let t = TempDir::new().unwrap();
+
+    #[cfg(not(windows))]
+    let script_code = indoc! { r#"cargo build --release && mkdir -p ../scarb/cairo-plugin && cp target/release/libfoo.so ../scarb/cairo-plugin"#};
+
+    #[cfg(windows)]
+    let script_code = indoc! { r#"cargo build --release && mkdir -p ../scarb/cairo-plugin && cp target/release/libfoo.dll ../scarb/cairo-plugin"#};
+
+    CairoPluginProjectBuilder::start()
+        .name("foo")
+        .scarb_project(|b| {
+            b.name("foo")
+                .version("1.0.0")
+                .manifest_package_extra(formatdoc! {r#"
+      include = ["Cargo.lock", "Cargo.toml", "src"]
```

Comment:
I'm not sure it's supposed to work that way 🤔 

These files are included in the tarball by default, shouldn't we use all the files that are included in the tarball by default when running a package script?

Originally posted in: https://github.com/software-mansion/scarb/pull/1789#discussion_r1877863937